antora-playbook.yml

```diff
@@ -35,7 +35,7 @@ content:
     - url: https://github.com/apache/camel-k.git
       branches:
         - main
-        - release-1.4.x
+        - release-1.6.x
```

Comment:
We can't build two branches with the same version, `latest` in this case is the version both in `main` and in `release-1.6.x`. This needs to be set to `1.6.x`:

https://github.com/apache/camel-k/blob/f23eecc4f57d78348abb8b5ba543278f000d2dcd/docs/antora.yml#L3

Reply:
Thanks Zoran, I have fixed this in camel-k